sysutils/beanstalkd/Portfile

```diff
@@ -19,19 +19,17 @@ long_description        beanstalkd is a fast, distributed, in-memory workqueue \
                         high-volume web applications by running most \
                         time-consuming tasks asynchronously.
 
-checksums               rmd160  a5ac5261b14092b087747500324753df01c11f26 \
-                        sha256  9d8119ac782c2d699842020fd6776eca21afa199b046884bbb2f3e01037b1026
-
-patchfiles              patch-Makefile.diff
+checksums               rmd160  744239013f44df8a8ac8aca17b31ffed5521f34b \
```

Comment:
We usually nowadays also add the `size` field to checksums, so the next time you update this port it would be great if you could add it.

`port lint --nitpick` also tells you about this, and the field can be generated by removing the checksums line completely and running `port -v checksum`.